spec.html

```diff
@@ -26748,19 +26748,19 @@ <h1>Date Time String Format</h1>
 THH:mm:ss.sss
         </pre>
         <p>All numbers must be base 10. If the `MM` or `DD` fields are absent `"01"` is used as the value. If the `HH`, `mm`, or `ss` fields are absent `"00"` is used as the value and the value of an absent `sss` field is `"000"`. When the time zone offset is absent, date-only forms are interpreted as a UTC time and date-time forms are interpreted as a local time.</p>
-        <p>Illegal values (out-of-bounds as well as syntax errors) in a format string means that the format string is not a valid instance of this format.</p>
+        <p>Illegal values (out-of-bounds as well as syntax errors) in a format string mean that the format string is not a valid instance of this format.</p>
         <emu-note>
-          <p>As every day both starts and ends with midnight, the two notations `00:00` and `24:00` are available to distinguish the two midnights that can be associated with one date. This means that the following two notations refer to exactly the same point in time: `1995-02-04T24:00` and `1995-02-05T00:00`</p>
+          <p>As every day both starts and ends with midnight, the two notations `00:00` and `24:00` are available to distinguish the two midnights that can be associated with one date. This means that the following two notations refer to exactly the same point in time: `1995-02-04T24:00` and `1995-02-05T00:00`. ISO 8601 allows an hour of 24 only to indicate the end of a calendar day within a time interval, but ECMAScript permits it in representations of single time points.</p>
```

Comment:
I find this addition here confusing personally. "Hour of 24" is hard to parse, it's not clear that "time interval" is not an ES thing but refers to an ISO8601 concept, and "single time points" could also be confused for an ES term. Suggestions:

* Use "an hour string of `24`" or similar
* Use "point in time" over "time points"

I'm struggling to understand the relevance of this note, though. @gibson042 can you say more about what you're trying to clarify?

Reply:
It's intended to highlight how ECMAScript is coloring outside the lines by allowing strings that ISO 8601 does not. I'll reword.